.github/workflows/title.yml

```diff
@@ -0,0 +1,19 @@
+name: PR Validation 
+on:
+  pull_request:
+    types: [opened, edited, synchronize, reopened]
+
+jobs:
+  title-validate:
+    name: Title
+    runs-on: ubuntu-latest
+    steps:
+    - uses: deepakputhraya/action-pr-title@master
+      with:
+        regex: '(Fixes |Refs )\d+: .+' # Regex the title should match.
```

Comment:
Right now we are using CONTENT-41 for example in the commit messages/PR's in the front-end.  This proposes that we change this to Katello's syntax correct?

Reply:
Yep, this was sorta decided the week that you were out, the reason is that we wanted to associate more than one pr with an issue, and be able to indicate whether the pr actually 'fixes' the issue, or is a secondary change